src/resolvers/TableViewInvitationResolver.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable indent */
+import { Invitation } from '../models/invitation.model'
+import { IntegerType, ObjectId } from 'mongodb'
+import { Context } from '../context'
+import { ApolloError } from 'apollo-server'
+import mongoose, { Error } from 'mongoose'
+import { checkUserLoggedIn } from '../helpers/user.helpers'
+import { User } from '../models/user'
+import { GraphQLError } from 'graphql'
+
+const TableViewInvitationResolver = {
+  Query: {
+    async getInvitations(parent: any, args: { query: string, limit: number, offset: number }, context: any) {
+      // const { userId, role } = (await checkUserLoggedIn(context))(['admin', 'manager']);
+      try {
+        const { query } = args;
+        const limit = args.limit ?? 10;
+        const offset = args.offset ?? 0;
+
+        console.log(`Querying for query: ${query}`);
+    
+        if (!query) throw new GraphQLError('No query provided');
+    
+    const searchCriteria = {
+      $or: [
+        { 'invitees.email': { $regex: query, $options: 'i' } },
+        { 'invitees.name': { $regex: query, $options: 'i' } },
```

Comment:
Additionally, in other implementations, the name will not be included in the invitations.

Reply:
> Additionally, in other implementations, the name will not be included in the invitations.

@Oliviier-dev first you have to undersand the codes.

Thanks